### PR TITLE
Update documentation

### PR DESCRIPTION
**Pull request by ain**

### CODE REVIEW

**Comment by ain** [comment]
@davleh please review. No functionality change.

File: README.md [hunk 1]
```diff
@@ -77,47 +77,26 @@ data-accordion-panel-activated | boolean     | Denotes active panel opened on pl
...
-- `grunt qa` run QA task that includes tests and JSHint
-- `grunt build` minify source to dist/
+### Emitted events
+
+Event                      | Emitter                | Description
+:-----                     | :--------              | :-----------
+`initialized.iptAccordion` | wrapper / base element | On successful plugin initialization              |
+`beforeOpen.iptAccordion`  | individual panel       | Before panel is opened                           |
+`afterOpen.iptAccordion`   | individual panel       | After panel has been opened (animation complete) |
+`afterOpen.iptAccordion`   | individual panel       | After panel has been opened (animation complete) |
```
Comment (davleh) [inline README.md]: @ain duplicated lines for afterOpen and beforeClose event

**Comment by davleh** [comment]
@ain only one small thing in the event documentation as commented

**Comment by ain** [comment]
@davleh thanks for the review. Back at you :)

**Pull request merged**